### DISCUSSION TITLE
Rlink 2 V9 Phone button not working in Carplay

### DISCUSSION THREAD

**Discussion by sdade86** [cedricp/ddt4all#1090] [Q&A]
Good morning! Thank you for help, finally issue with sleep mode has been solved. I've deactivated parameter CF_Passing_inhibition according to backup dumps. R-link has been upgraded to 9.x without any issues as well.

_Originally posted by @DmitriGoncharick in https://github.com/cedricp/ddt4all/issues/774#issuecomment-1427571704_

@DmitriGoncharick Hello mate, i have some question about V9, maybe you can help me.

    I have update to V9 and using Carplay the phone button on steering whell not working for aswer/hung up calls. Do you have the same problem?
    If i want to downgrade to V8 is it possible? or downgrade is only to V7?
    Where you have found LGU for rollback to V7? need any special procedure or only put in USB stick and install?

Thanks in advance!

**Comment by avtovaz1** [comment]
Hello guys, does anyone have an update on the issue? On V7 this button was working properly for answering/hanging phone calls on both Android and Carplay.
